### ISSUE TITLE
Latex build ignores highlight_language

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#409]
When building a document with -b latex, the highlight_language config option seems to be ignored.

Take the following document:

```
#!rst
::

 #include <stdio.h>
 int main()
 {
    return 0;
 }
```

I set "highlight_language" in conf.py:

highlight_language = 'c'

and run:

sphinx-build -b latex -d build/doctrees . build/latex/

The output is not highlighted (although it is highlighted in the html output). However, if I explicitly ask for highlighting in the document, by adding:

```
#!rst
.. highlight:: c
```

Then it will show up highlighted correctly in the Latex output.

So there is a problem with the highlight_language option and Latex, not highlighting in general.

Using Sphinx version 0.6.4 on Ubuntu 10.04 with pdfTeX 3.1415926-1.40.10-2.2

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/409
- Originally reported by: Matt Giuca
- Originally created at: 2010-05-07T11:24:12.027

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-05-22 09:09:07+00:00_

Thanks for the report!  This is now fixed in <<cset 015c1c13a5448ddee0cb4e240ebb1f2a6eca15fa>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:18+00:00_

Removing version: 0.6.4 (automated comment)
